statsmodels/tsa/base/tsa_model.py

```diff
@@ -71,6 +71,14 @@ def _init_dates(self, dates, freq):
         self.data.dates = dates
         self.data.freq = freq
 
+        # Test for nanoseconds in early pandas versions
+        if _freq_to_pandas[freq].freqstr == 'N':
+            from pandas import version
+            if version.version < '0.14':
+                raise NotImplementedError('Nanosecond index not available in'
+                                          ' Pandas < 0.14')
```

Comment:
This looks kind of like a hack, but I have not been able to find a simple way to test for the presence of the nanosecond bug itself. if anyone has a better way, that would be great.